package.json

```diff
@@ -44,7 +44,7 @@
     "url": "https://opencollective.com/openlayers"
   },
   "dependencies": {
-    "color-name": "^2.0.0",
+    "color-rgba": "^2.4.0",
```

Comment:
The `color-rgba` package depends on `color-name`.  So the behavior with named colors is retained from #15128..